install_config/aggregate_logging.adoc

```diff
@@ -1875,39 +1812,68 @@ systemctl restart docker
 
 . Update the xref:fluentd-upgrade-source[Fluentd log source].
```

Comment:
Hey @bmcelvee - should this link to fluentd-upgrade-source still stay in this file? It points to the section heading that is just next (and deleted). The other xrefs to fluentd-upgrade-source have been deleted from this file, so I think that may have remained in error. But if we remove it, does the rest of the text still make sense?

Reply:
Thanks for catching that @vikram-redhat! I opened a follow-up PR: https://github.com/openshift/openshift-docs/pull/8303. I also clicked through all of the links in the topic to make sure they're correct.

There's one that points to manual upgrades in the following section: 
https://docs.openshift.com/container-platform/3.7/install_config/aggregate_logging.html#aggregate-logging-performing-elasticsearch-maintenance-operations

Should I update it to point to automated upgrades? (The link still works)

Reply:
Thanks @bmcelvee for the fix. I am not sure about the manual upgrades link. I am not sure if that is remaining for 3.9. If it is not remaining, then yeah you should point to the automatic upgrades.